src/utils/export-map.ts

```diff
@@ -2,9 +2,9 @@ import fs from 'node:fs'
 import path from 'node:path'
 
 import type { TSESLint, TSESTree } from '@typescript-eslint/utils'
+import { parse as parseComment_ } from 'comment-parser'
```

Comment:
I am wondering, is it possible to lazyload `comment-parser` to improve the cold start performance? It is only used in `no-deprecated`, and we might be able to speed up performance a bit.

Reply:
By `cjsRequire`?

```ts
let parseComment_: typeof parseCommentType | undefined

const parseComment = (comment: string): Block => {
  parseComment_ ??=
    // eslint-disable-next-line @typescript-eslint/consistent-type-imports
    cjsRequire<typeof import('comment-parser')>('comment-parser').parse
  const restored = `/**${comment.split('\n').reduce((acc, line) => {
    line = line.trim()
    return line && line !== '*' ? acc + '\n  ' + line : acc
  }, '')}
  */`
  const [doc] = parseComment_(restored)
  return {
    ...doc,
    tags: doc.tags.map(t =>
      t.name && fixup.has(t.tag)
        ? { ...t, description: `${t.name} ${t.description}` }
        : t,
    ),
  }
}
```

Reply:
Here we go: https://[github.com/un-ts/eslint-plugin-import-x/pull/314/commits/23d0e0881da352556f5a8176422dd6dcc54ed51b](https://github.com/un-ts/eslint-plugin-import-x/pull/314/commits/23d0e0881da352556f5a8176422dd6dcc54ed51b)